wgpu/src/shader/color/oklab.wgsl

```diff
@@ -20,7 +20,7 @@ fn interpolate_color(from_: vec4<f32>, to_: vec4<f32>, factor: f32) -> vec4<f32>
     var color = to_rgb * (mixed * mixed * mixed);
 
     // Alpha interpolation
-    color.a = mix(from_.a, to_.a, factor);
+    color = to_ * factor + from_ * (1.0 - factor);
```

Comment:
Pretty sure you could just call `color = mix(from_, to_, factor)` here instead, but maybe this is more clear that we are lerping?